install_with_docker.sh

```diff
@@ -105,57 +125,97 @@ install_docker() {
             "deb [arch=$arch] https://download.docker.com/linux/$os $(lsb_release -cs) stable"
         $apt_cmd update
         echo "Installing docker"
-        $apt_cmd install docker-ce docker-ce-cli containerd.io || print_error_and_exit "Docker installation failed. Please install docker manually and re-run the command."
+        $apt_cmd install docker-ce docker-ce-cli containerd.io || {
+            post_event "install_failed" "install_docker: Docker installation failed during apt-get install on $os"
+            print_error_and_exit "Docker installation failed. Please install docker manually and re-run the command."
```

Comment:
make the msg for this error line and above post_event to be exact same thing, since they both need to convey the same thing